corehq/apps/locations/static/locations/spec/location_drilldown_spec.js

```diff
@@ -1,9 +1,6 @@
 /*eslint-env mocha */
-/*global _ LocationSelectViewModel */
 
-
-/*eslint no-unused-vars:0*/
-var LOAD_LOCS_URL = '/locations/api/';
+var LocationSelectViewModel = hqImport('locations/js/location_drilldown').LocationSelectViewModel;
```

Comment:
would it work to move this inside the `describe` block below so as not to assign it to a global variable?

Reply:
Yup, that should work. I had just been mirroring where it was before